pkg/transferer/encryption.go

```diff
@@ -29,6 +31,20 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
+const (
+	iteration    int = 10000 // Openssl makes this number of iterations by default when encrypting/decrypting with pbdkf2
+	split        int = 48    // Number of bytes needed to get key and iv from pbkdf2. 32 on key and rest on iv
+	saltSize     int = 8     // Salt size by default in openssl
+	passwordSize int = 16    // Size of password in bytes when generating random password
+)
+
+var gzw *gzip.Writer
+var tw *tar.Writer
+var writer *cipher.StreamWriter
+var gzr *gzip.Reader
+var tr *tar.Reader
+var reader *cipher.StreamReader
```

Comment:
Please don't use global variables, there is no need for them. You can create a wrapper struct for them if you need

For example, I suggest implementing new structs in the `dump` package to use instead of `tar.Writer` and `tar.Reader` across the code